README.md

```diff
@@ -30,3 +30,51 @@ function mystery(n) {
 Add your answer to this markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
+
+When n <= 1, the function does not perform any operation and returns, and the time complexity is O(1)
+
+        for(var i = 0; i < n*n; i++) { //n^2
+            for(var j = 0; j < n; j++) { // n
+                for(var k = 0; k < n*n; k++) { // n^2
+                    count = count + 1; // 1
+        mystery(n / 3)mystery(n / 3)mystery(n / 3) // *3 and each time recursive the input size will be n to n/3
+
+$total loop = n^2*n*n^2 =n^5 operations $
+
+T(n) represents the function running time when the input is n. T(n) = O(1)
```

Comment:
Incorrect notation. A T(n) expression cannot be equal to an asymptotic expression.